lib/avm_api.rb

```diff
@@ -4,7 +4,7 @@ def initialize(host:)
   end
 
   def validate_parameters(document, content, mimetype)
-    response = Faraday.post(url('/parameters/validate'), {
+    response = Faraday.post(url('/visualization'), {
```

Comment:
pri tom /validate endpointe sa validujú iba serverové parametre, ale nepríde sa na všetky ďalšie problémy. Validácia by mala byť na jednom mieste, ale to prekopeme v konsolidácii Autogram core komponentu neskôr. Tento fix zatiaľ pomôže v tom, že klientovi nedovolíme ani uploadnúť dokument, ak je zlý - doteraz to padlo až pri vizualizácii.